source/npm/qsharp/ux/qsharp-circuit.css

```diff
@@ -72,6 +72,34 @@
     fill: var(--vscode-button-secondaryForeground, #ffffff);
   }
 
+  a.qs-circuit-source-link .qs-qubit-label {
+    pointer-events: auto;
+  }
+
+  /* Hover effect for clickable gates */
+  a.qs-circuit-source-link:hover .gate-unitary {
+    fill: var(--vscode-button-secondaryHoverBackground, #4d4d4d);
```

Comment:
When picking default values for these, the thing to keep in mind is that (I believe) all the out-of-the-box VS Code themes defined all these variables _except_ the two high contrast themes (which happen to be the ones I use 😊). We don't really need to go out of our way to make sure the defaults look good with custom themes, because people do wacky things, but we should make sure the defaults work well with the out-of-the-box high contrast themes.
Checking those, under the light high contrast theme, the unitary gates are not readable with these colors, and the rest of the elements don't change. Under the dark high contrast theme, most things are fine, but measurements and kets don't change any color.

Reply:
That's an excellent point, thank you. I'll test with those themes

Reply:
Dang, HC themes can be tough to work with. Maybe the extra requirements placed on them for accessibility means there are more undefined fields? Not sure, but in any case we can fiddle with the CSS later if we decide things look bad enough to change it. Thanks for looking into this!